README.md

```diff
@@ -12,15 +12,42 @@ work!
 - [Drush](https://github.com/drush-ops/drush-launcher#installation---phar)
 - SQLite and the related PHP extensions
 
-## Usage
+## Initializing a new project
 
-To create a new Tome project, run:
+To create a new Tome project in the `my_site` subdirectory, run:
+
+```bash
+mkdir my_site
+cd my_site
+composer create-project drupal-composer/drupal-project:8.x-dev . # <-- the dot at the end means current directory
+composer require drupal-tome/tome-project
```

Comment:
Composer projects are never added as dependencies - they're just templates for new projects. The previous instructions were correct here.